### ISSUE TITLE
ERROR: realloc(): invalid pointer Abort core dumped

### ISSUE THREAD

**Issue by akashdave-v** [AlexeyAB/darknet#3915]
I got this ERROR while training and it stopped suddenly after 89 iterations. I have a small dataset of around 300 images. I am training YOLOv3 tiny. Also, on and after 82 iterations avg. loss is 'nan'.

I had configured custom cfg file as below. 
`batch=64,
subdivison=8,
learning_rate=0.00025,
burn_in=1000,
max_batches=6000,
steps=4800, 5400`

Logs are as below-
[train2.log](https://github.com/AlexeyAB/darknet/files/3609702/train2.log)

@AlexeyAB Please any help would be appreciated.

**Comment by akashdave-v** [comment]
??
